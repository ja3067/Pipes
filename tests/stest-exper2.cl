x = 3

if x == 3:
	y : int = 4
