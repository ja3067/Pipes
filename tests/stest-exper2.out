SSyntaxError: cannot explicitly type variable 'y' while in conditional branches
